src/main/java/org/magicdgs/readtools/utils/read/stats/pairstat/PairIntegerTagCounter.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Daniel Gomez-Sanchez
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package org.magicdgs.readtools.utils.read.stats.pairstat;
+
+import org.magicdgs.readtools.utils.math.RelationalOperator;
+import org.magicdgs.readtools.utils.read.stats.PairEndReadStatFunction;
+import org.magicdgs.readtools.utils.read.stats.StatFunctionUtils;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+
+/**
+ * Counts the number of read pairs whose integer SAM tag is compared to a threshold.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+// TODO: add group for this and documented feature once is a plugin
+public class PairIntegerTagCounter implements PairEndReadStatFunction<Integer, Boolean> {
+
+    // TODO: maybe this could be more liek a TaggedArgument and then initialize the counters
+    // TODO: something like --count-pair-int-tag:NM:EQ 2
```

Comment:
Open issue for this or add to an epic one.

Reply:
Removed from here - this should be for `PairIntegerTagListCounter` (and mentioning #449 there)